cloudify_agent/shell/commands/daemons.py

```diff
@@ -161,6 +157,11 @@ def create(**params):
     custom_arg = attributes.pop('custom_options', ())
     attributes.update(_parse_custom_options(custom_arg))
     click.echo('Creating...')
+    manager_ip = get_manager_ip(attributes)
```

Comment:
will be nicer if it will be more specific, receive manager_ips and agent_ip as arguments, and not obscure "attributes" argument